### DISCUSSION TITLE
Emails alerts in nocodb? How do people manage this ...

### DISCUSSION THREAD

**Discussion by ReessKennedy** [nocodb/nocodb#8880] [Q&A]
How do folks handle email alerts for a base or table? Like: What if I want to get an email each time a new row is added to table X i a base? Tools like Baserow have a built in notification system but it does not look like Nocodb has anything like this other than the cool audit view, which I like, but this is like a Godview. 

Does everyone just write a trigger in something like Zapier or n8n that goes to SMTP for email alerts?

**Comment by o1lab** [comment]
At the moment, you can [create a webhook](https://docs.nocodb.com/automation/webhook/create-webhook/) everytime a row gets added to a table.
